libp2p/peerinfo.nim

```diff
@@ -105,9 +105,13 @@ proc publicKey*(p: PeerInfo): Option[PublicKey] {.inline.} =
     result = some(p.privateKey.getKey())
 
 proc id*(p: PeerInfo): string {.inline.} =
-  result = p.peerId.pretty()
+  if not isNil(p):
```

Comment:
Not sure why p would be `Nil`, @cheatfate is this related to your changes by any chance?

Reply:
seems like this was related to an earlier change that has since been removed.